presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -354,6 +354,30 @@ Optimizer Properties
     The single distinct optimization will try to replace multiple ``DISTINCT`` clauses
     with a single ``GROUP BY`` clause, which can be substantially faster to execute.
 
+``optimizer.push-aggregation-through-join``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``Boolean``
+    * **Default value:** ``true``
+    * **Description:**
+
+    When an aggregation is above an outer join and all columns from the outer side of the join
+    are in the grouping clause, the aggregation is pushed below the outer join. This optimization
+    is particularly useful for correlated scalar subqueries, which get rewritten to an aggregation
+    over an outer join. For example:
+
+    .. code-block:: sql
```

Comment:
Note that the default code style is SQL, so we can use the shorthand (no need to declare the code block type):
```
For example::

    SELECT ...